internal/k8s-engine/policy/error.go

```diff
@@ -2,4 +2,5 @@ package policy
 
 import "github.com/pkg/errors"
 
+// ErrPolicyConfigMapNotFound returns an error indicating that Policy cannot be found.
```

Comment:
Suggestion: 🤔 Maybe `is`/`defines` is better in this context?

```suggestion
// ErrPolicyConfigMapNotFound defines an error indicating that Policy cannot be found.
```

Reply:
yep, let's stay with `defines` as it is used already in other places